src/core/ddsc/include/dds/dds.h

```diff
@@ -4682,6 +4682,29 @@ DDS_EXPORT dds_return_t
 dds_free_typeinfo (
   dds_typeinfo_t *type_info);
 
+
+/**
+ * @brief Gets the sertype of an entity
+ *
+ * The provided entity must be a topic or endpoint. This function
+ * returns the sertype of the entity.
```

Comment:
I think it would be wise to note that it doesn't increment the refcount of the sertype and that the lifetime of the returned pointer is at least that of the lifetime of the entity on which it was invoked. (Just like `create_topic_sertype` really.)